bluebrain/sysconfig/bluebrain5/modules.yaml

```diff
@@ -13,6 +13,11 @@ modules:
       hash_length: 0
       projections:
         all: '{name}/{version}'
+        neurodamus-models model=hippocampus: 'neurodamus-hippocampus/{version}'
+        neurodamus-models model=neocortex: 'neurodamus-neocortex/{version}'
+        neurodamus-models model=neocortex +plasticity: 'neurodamus-neocortex-plasticity/{version}'
+        neurodamus-models model=neocortex +ngv+metabolism: 'neurodamus-neocortex-multiscale/{version}'
```

Comment:
It seems this PR created modules 
```
neurodamus-models-multiscale/6.7.4
neurodamus-models-plasticity/6.7.4
```
Why not 
```
neurodamus-neocortex-multiscale/6.7.4
neurodamus-neocortex-plasticity/6.7.4
```
?

Reply:
Thanks for noticing! Indeed, I probably have to fix the projection better.

Reply:
OK, the `applications_hpc` overwrote the projections I had defined. Should be fixed now.